peewee.py

```diff
@@ -1586,6 +1601,8 @@ def get_parse_map(self):
         }
 
     def quote(self, s):
+        if re.match(r'[a-zA-Z]\w*$', s) and s.upper() not in self.sql_reserved_words:
+          return s
```

Comment:
I think this is going to negatively impact performance. I'd be curious to see some profiling before and after.